lib/models/jellyfin_models.dart

```diff
@@ -2115,9 +2115,9 @@ class BaseItemDto with RunTimeTickDuration {
   @HiveField(130)
   int? albumCount;
 
-  /// Gets or sets the song count.
+  /// Gets or sets the track count.
   @HiveField(131)
-  int? songCount;
+  int? trackCount;
```

Comment:
we can't change this one. the variable has to be the same as in the Jellyfin API (except for capitalization), so that the automatic serialization and deserialization works.  
Line 1586 is affected as well.

The other changes should be fine, as they really are only internal.